compose-builder/add-security.yml

```diff
@@ -91,6 +91,7 @@ services:
       - edgex-init:/edgex-init:ro,z
       - vault-config:/vault/config:z
       - /tmp/edgex/secrets:/tmp/edgex/secrets:z
+      - kong:/tmp/kong
```

Comment:
add :z
```suggestion
      - kong:/tmp/kong:z
```

Reply:
Ok - added that - what does that option do again?

Reply:
it is to make this shareable across containers in the SELinux-enabled regression test environment